ios/TeamHome/TeamHome/Models/Mock json/UserExample.json

```diff
@@ -0,0 +1,12 @@
+ {
```

Comment:
Property names in JSON files should be double quoted. This isn't properly formed JSON.